doc/interfaces/dg.rst

```diff
@@ -0,0 +1,23 @@
+DG Interface
+================
+
+Interface to `DG Technologies <http://dgtech.com>`__ Beacon
+
+
+Bus
+---
+
+.. autoclass:: can.interfaces.dg.dg.DGBus
+
+
+Capability
+-------------
+The ``DGBus`` class implements several python-can methods::
+
+    send, send_periodic, recv, set_filters, flush_tx_buffer, _detect_available_configs, shutdown
```

Comment:
I think there is no need to mention the methods with definitely need to be implemented anyways. It raises the question if other interfaces do it even if they do not mention it. I'd suggest including these:

```suggestion
The ``DGBus`` class implements several optional methods: ``send_periodic`` and ``set_filters`` within the driver, as well as ``flush_tx_buffer`` and ``_detect_available_configs``.
```